Mathlib/Dynamics/BirkhoffSum/Pointwise.lean

```diff
@@ -0,0 +1,403 @@
+/-
+Copyright (c) 2025 Lua Viana Reis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Lua Viana Reis, Oliver Butterley
+-/
+module
+
+public import Mathlib.Algebra.Order.Ring.Star
+public import Mathlib.Algebra.Order.SuccPred.PartialSups
+public import Mathlib.Algebra.Order.Group.PartialSups
+public import Mathlib.Analysis.SpecialFunctions.Log.ENNRealLogExp
+public import Mathlib.Data.Real.StarOrdered
+public import Mathlib.Dynamics.BirkhoffSum.QuasiMeasurePreserving
+public import Mathlib.GroupTheory.MonoidLocalization.Basic
+public import Mathlib.MeasureTheory.Constructions.Polish.Basic
+public import Mathlib.MeasureTheory.Function.ConditionalExpectation.Basic
+public import Mathlib.MeasureTheory.Integral.DominatedConvergence
+public import Mathlib.MeasureTheory.MeasurableSpace.Invariants
+public import Mathlib.Topology.EMetricSpace.Paracompact
+public import Mathlib.Topology.Separation.CompletelyRegular
```

Comment:
Can you remove some of these imports (some of these are certainly implied by other imports)? To trim the imports, just try to remove a line, see if the file still compiles, add it back if it doesn't, and iterate.